tests/cases/fourslash/importNameCodeFixNewImportFile5.ts

```diff
@@ -0,0 +1,16 @@
+/// <reference path="fourslash.ts" />
+
+//// [|bar/*0*/();|]
+
+// @Filename: foo.ts
+//// interface MyStatic {
+////     bar(): void;

```

Comment:
If I'm reading the code correctly, it seems like the case that distinguishes the two `tryGetMemberInModuleExports*` call-sites is a property named "default".  Would it be interesting to test that explicitly.